deploy/kubernetes/releases/csi-digitalocean-dev.yaml

```diff
@@ -1,17 +1,17 @@
-# Copyright 2018 DigitalOcean
-# 
+# Copyright 2019 DigitalOcean
```

Comment:
Github doesn't support commenting on lines outside the context of a change, so I'm noting it down here:

Could you please update the API versions of our StatefulSet and DaemonSet manifests from `apps/v1beta2` to `apps/v1` in this file? The former are deprecated in Kubernetes 1.16 and wouldn't work anymore. (See also #202.)

I have already made a note to also update the non-dev release manifests when we do the next release. I'll take care of that.